src/types/03-npm.js

```diff
@@ -1,68 +1,73 @@
 const npm = require('concierge/npm'),
     path = require('path'),
-    fs = require('fs-extra');
+    files = require('concierge/files');
 
-const fixMain = (jsonPath, property) => {
-    const json = JSON.parse(fs.readFileSync(jsonPath).toString());
+const copy = async(src, dst) => {
+    return await files.writeFile(dst, await files.readFile(src));
+};
+
+const fixMain = async(jsonPath, property) => {
+    const json = await files.readJson(jsonPath);
     json[property] = 'index.js';
-    fs.writeFileSync(jsonPath, JSON.stringify(json, null, 4));
+    return await files.writeFile(jsonPath, JSON.stringify(json, null, 4));
 };
 
-const rebuildModule = (pack, dir) => {
+const rebuildModule = async(pack, dir) => {
     const packageJsonPath = path.join(dir, 'package.json');
-    try {
-        fs.unlinkSync(packageJsonPath);
+    if (await files.fileExists(packageJsonPath) === 'file') {
+        await files.unlink(packageJsonPath);
     }
-    catch(e) {}
-    fs.copySync(path.join(dir, `node_modules/${pack}/package.json`), packageJsonPath);
-    fixMain(packageJsonPath, 'main');
+
+    await copy(path.join(dir, `node_modules/${pack}/package.json`), packageJsonPath);
+    await fixMain(packageJsonPath, 'main');
 
     try {
         const kassyJsonPath = path.join(dir, 'kassy.json');
-        try {
-            fs.unlinkSync(kassyJsonPath);
+        if (await files.fileExists(kassyJsonPath)) {
+            await files.unlink(kassyJsonPath);
         }
-        catch(e) {}
-        fs.copySync(path.join(dir, `node_modules/${pack}/kassy.json`), kassyJsonPath);
-        fixMain(kassyJsonPath, 'startup');
+        await copy(path.join(dir, `node_modules/${pack}/kassy.json`), kassyJsonPath);
+        await fixMain(kassyJsonPath, 'startup');
     }
     catch(e) {}
```

Comment:
And log what exactly? This is a copy if exists operation, we don't really care if it fails as long as it tried